### ISSUE TITLE
Useful pinned messages link gets unintuitively buried in new UI

### ISSUE THREAD

**Issue by pfunks** [RocketChat/Rocket.Chat#9648]
### Description:

In the new Rocket.Chat UI, not sure if it is just the current 0.61 version or if it goes further back, the icon to access a channel's pinned messages has been buried/hidden under the "three dots" menu in the upper right.

In my experience with using Rocket.Chat, pinned messages are important for things like channel rules or guidelines, and institutional memory. So I think it's important enough to put the pinned message link alongside these:
![image](https://user-images.githubusercontent.com/22420130/36011672-90906da2-0d1e-11e8-869d-c27f38d29aed.png)

...and not buried in this unintuitive menu:

![image](https://user-images.githubusercontent.com/22420130/36011764-246a0cae-0d1f-11e8-9e38-9ec7d6d9c73b.png)

### Server Setup Information:

* Version of Rocket.Chat Server:
0.61.0

**Comment by sciborg** [comment]
Yes.. Please consider moving the pinned messages icon alongside the others not hidden in that dropdown thats on far right
